intelmq/bots/parsers/bambenek/parser_c2dommasterlist.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+See docs/Feeds.md
+"""
+
+import sys
+
+from intelmq.lib.bot import ParserBot
+from intelmq.lib.message import Event
+
+
+class Bambenekc2dommasterlistParserBot(ParserBot):
+
+    def parse_line(self, line, report):
+        if line.startswith('#'):
+            self.tempdata.append(line)
+        else:
+            lvalue = line.split(',')
+            event = Event(report)
+
+            event.add('source.fqdn', lvalue[0])
+            event.add('event_description.text', lvalue[1])
+            event.add('time.source', lvalue[2] + " UTC")
+            event.add('event_description.url', lvalue[3])
+            event.add('classification.type', 'c&c')
+            event.add('status', 'online')
```

Comment:
yes, according to the DHO that's for what it is. However, we have not clearly defined valid values in the DHO for this field ;-) Means: we might have to refactor this later.... But it's ok for now IMHO.